### PR TITLE
Update news and bump version number

### PR DESCRIPTION
**Pull request by dmcglinn**
I feel like there was also something to do with Rstudio in the updates but I couldn't quickly spot it in the closed PR's. Thanks!

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+
+NEW PACKAGE NAME
+
+* The `EcoData Retriever` has been renamed to `Data Retriever` to reflect its 
+utility outside of ecological data and consequently we have renamed the R package
+from `ecoretriever` to `rdataretriever`
+
+NEW COAUTHORS
+
+* We welcome Henry Senyondo and Shawn Taylor as coauthors on the package. Thanks
```
Comment (henrykironde) [inline NEWS.md]: Should it be co-authors?

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+
+* Accomodate new retriever naming conventions in fetch
+* Don't change the class or return the update log
+* Specify in documentation which functions are for internal use.
+* Change dataset names in source and README.md
+
+BUG FIXES
+
+* Search for Anaconda installs of the `Data Retriever`
+
```
Comment (ethanwhite) [inline NEWS.md]: Add: Obtain correct home path in RStudio on Windows

File: NEWS.md [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+NEW COAUTHORS
+
+* We welcome Henry Senyondo and Shawn Taylor as coauthors on the package. Thanks
+for the great help Henry and Shawn!
+
+NEW FEATURES
+
+* Add `reset` which allows a user to delete all the `Data Retriever` downloaded
+files
+
```
Comment (ethanwhite) [inline NEWS.md]: Add: Add `json` and `xml` as output options

File: NEWS.md (hunk 1 above)
Comment (henrykironde) [inline NEWS.md]: I guess coauthor is fine. :)

File: NEWS.md (hunk 3 above)
Comment (dmcglinn) [inline NEWS.md]: done.

File: NEWS.md (hunk 2 above)
Comment (dmcglinn) [inline NEWS.md]: done.

**Comment by dmcglinn** [comment]
I made the suggested changes so I'm merging now. I'll begin the CRAN submission process and I'll let you know how it goes.

**Pull request merged**